documentation/docs/known-issues.md

```diff
@@ -15,25 +15,29 @@ FlashList and FlatList have very different internal. While the API is almost the
 
 Please note most lists do mount with deterministic sizes so make sure to check if you really need workarounds.
 
-### 2) `onEndReached` event doesn't have `distanceFromEnd`
+### 2) `useState` values will be recycled
+
+If your component hierarchy returned from `renderItem` makes use of `useState`, you will find that state values set by components in other cells of the list may be reflected in cells that use the recycled component. There are two workarounds for this, see [the Github issue](https://github.com/Shopify/flash-list/issues/527#issuecomment-1184463150).
```

Comment:
I don't expect us to solve it as it's expected. I think it's worth creating a dedicated page in `Fundamentals` called e.g. `Using state` where we would detail potential workarounds and reasoning behind it. In this case, it might be better if we write it up most likely - wdyt @naqvitalha?

Reply:
that sounds great!

took a stab at roughing in a smaller blurb for you in the interim